Tutorial.md

```diff
@@ -11,477 +10,249 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
+                "choices": [
+                    {
+                        "choice": "Swift",
+                        "url": "/questions/1/choices/1",
+                        "votes": 2048
+                    }, {
+                        "choice": "Python",
+                        "url": "/questions/1/choices/2",
+                        "votes": 1024
+                    }, {
+                        "choice": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "choice": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
                     }
-                }
-
-A response usually bears some payload sent to our client. Ideally it details the resource in question. In this case our response representation is of the [`application/hal+json`][] media-type. Also in addition to an HTTP response message-body this response defines an HTTP response [message-headers][] in its payload.
-
-> **Note:** API Blueprint is **indentation sensitive**. See the [Note on Indentation][] for details.
->
-> **Note:** Specifying the media type in parenthesis after the response status codes generates implicit `Content-Type` HTTP header. Therefore you do not have to explicitly specify the `Content-Type` header.
->
-> **Note:** If your response does not need to define any additional headers (but `Content-Type`) you can skip the `Headers` section completely and write the `Body` section like so:
->
->     + Response 200 (application/hal+json)
->
->         {
->             "_links": {
->                 "self": { "href": "/" },
->                 "gists": { "href": "/gists?{since}", "templated": true }
->             }
->         }
+                ]
+            }
+        ]
+```
 
-## Complex Resource
-With the entry point of our API defined we can move forward. Since our API revolves around a Gist let's define its representation and an action to retrieve it:
+> **Note:** Specifying the media type after the response status code generates a `Content-Type` HTTP header. You do not have to explicitly specify the `Content-Type` header.
 
-    # Group Gist
-    Gist-related resources of *Gist Fox API*.
+The polls resource has a second action which allows you to create a new question. This action includes a description showing the structure you would send to the server to perform this action.
 
-    ## Gist [/gists/{id}]
-    A single Gist object. The Gist resource is the central resource in the Gist Fox API. It represents one paste - a single text note.
+```markdown
+### Create a New Question [POST]
 
-    The Gist resource has the following attributes:
+You may create your own question using this action. It takes a JSON object containing a question and a collection of answers in the form of choices.
 
-    - id
-    - created_at
-    - description
-    - content
+- question (string) - The question
```

Comment:
@kylef I am still uneasy with this. I understand your point on mixing `-`s and `+`s and I am fine with it. But I think we should have this unified in the tutorial to not confuse newcomers. 

Also, since we do not expect any upfront Markdown knowledge I would expect explanation of lists somewhere (-/+/*). Markdown list is a building stone of API Blueprint syntax..